custom/icds_reports/reports/__init__.py

```diff
@@ -25,13 +27,28 @@ def data_providers(self):
     @property
     def report_config(self):
         start_date, end_date = get_first_last_days(self.year, self.month)
+        month_before = start_date - relativedelta(months=1)
+        new_start_date = month_before.replace(day=21)
+        new_end_date = month_before.replace(day=20)
+        self.datespan.startdate = new_start_date
+        self.datespan.enddate = new_end_date
+        state = self.request.GET.get('icds_state_id', None)
```

Comment:
Hey @lwyszomi , just curious where are these state/district/block parameters being passed into GET request? I haven't been able to find it

Reply:
Hey @sravfeyn look like these parameters are not passed in GET, but in the newest code, we using location_id https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/reports/__init__.py#L41.

I think these 3 lines and conditions below we can remove because state/district/block are always None.